Fitness/Views/Cells/GymsCell.swift

```diff
@@ -69,39 +71,67 @@ class GymsCell: UICollectionViewCell {
     }
 
     // MARK: - CONSTRAINTS
-    func setupConstraints() {
+    private func setupConstraints() {
+        let descriptionLabelHieght = 15
+        let leadingPadding = 16
+        let trailingPadding = 4
+        
         colorBar.snp.updateConstraints { make in
             make.leading.equalToSuperview()
             make.height.equalToSuperview()
-            make.width.equalToSuperview().dividedBy(32)
+            make.centerY.equalToSuperview()
+            make.width.equalTo(5)
         }
 
         locationName.snp.updateConstraints { make in
-            make.top.equalToSuperview().offset(12)
-            make.leading.equalToSuperview().offset(15)
-            make.trailing.lessThanOrEqualToSuperview().inset(4)
+            make.top.equalToSuperview().offset(16)
+            make.leading.equalToSuperview().offset(leadingPadding)
+            make.height.equalTo(22)
+            make.trailing.lessThanOrEqualToSuperview().inset(trailingPadding)
         }
 
         status.snp.updateConstraints { make in
             make.leading.equalTo(locationName)
-            make.height.equalTo(15)
-            make.top.equalTo(locationName.snp.bottom).offset(2)
+            make.trailing.lessThanOrEqualToSuperview().inset(trailingPadding)
+            make.height.equalTo(descriptionLabelHieght)
+            make.top.equalTo(locationName.snp.bottom).offset(6)
         }
 
         hours.snp.updateConstraints { make in
-            make.leading.equalTo(status.snp.trailing).offset(5)
-            make.trailing.lessThanOrEqualToSuperview().inset(4)
-            make.height.top.equalTo(status)
+            make.leading.equalTo(status)
+            make.trailing.lessThanOrEqualToSuperview().inset(trailingPadding)
+            make.height.equalTo(descriptionLabelHieght)
+            make.top.equalTo(status.snp.bottom).offset(3)
         }
     }
 
     func setGymName(name: String) {
         locationName.text = name
     }
 
-    func setGymStatus(isOpen: Bool) {
-        let color: UIColor = isOpen ? .fitnessGreen : .fitnessRed
-        self.status.text = isOpen ? "Open" : "Closed"
+    func setGymStatus(isOpen: Bool, changingSoon: Bool) {
+        
+        var color: UIColor
+        var status: String
+        if isOpen {
+            if changingSoon {
+                color = .fitnessOrange
```

Comment:
instead of having a `var color` and setting it, i think we cna just do `self.status.textColor = .fitnessOrange`. same w/ status